source/_docs/z-wave.markdown

```diff
@@ -180,8 +180,25 @@ automation:
         scene_id: 11
 ```
 
+Some devices (like the HomeSeer wall switches) allow you to do things like double, and triple click the up and down buttons and fire an event.  These devices will also send `scene_data` to differentiate the events.  This is an example of double clicking the on/up button:
+
+```yaml
+# Example configuration.yaml automation entry
+automation
+  - alias: 'Dining room dimmer - double tap up'
+    trigger:
+      - event_type: zwave.scene_activated
+        platform: event
+        event_data:
+          entity_id: zwave.diningroomcans_level
```

Comment:
Leave off the "_level", as that's part of the value entity.  The entity_id sent with the event would just be ```zwave.diningroomcans``` in this case.

Reply:
These are things I would have had to learn as I was actually implementing it.  Thanks!